src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -51,7 +51,7 @@ export default class TableViewComponent extends Component {
         } else if (sortKey === SORTKEY_PRIORITY) {
             this.sortByPriority(items);
         } else {
-            this.setState((state) => ({ ...state, items }));
+            this.setState(() => ({ items }));
```

Comment:
We do not need callback here.

Reply:
See comment above: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/92#discussion_r202540313